iostat/iostat.go

```diff
@@ -172,21 +163,26 @@ func (iostat *IOSTAT) GetMetricTypes(_ plugin.ConfigType) ([]plugin.MetricType,
 	return mts, nil
 }
 
-//GetConfigPolicy
-func (iostat *IOSTAT) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
-	c := cpolicy.New()
-	return c, nil
+//GetConfigPolicy return configuration policy
+func (iostat *Iostat) GetConfigPolicy() (plugin.ConfigPolicy, error) {
+	c := plugin.NewConfigPolicy()
+	return *c, nil
+
 }
 
 // Init initializes iostat plugin
-func (iostat *IOSTAT) run(mts []plugin.MetricType) ([]string, map[string]float64, error) {
-	// TODO: reminder - remove these todo statements from the README (roadmap section) once
-	//       they are completed
-	// TODO: add validation that we are running sysstat version 10.2.0 or greater else print a
-	//       a warning or otherwise communicate to the user that they are potentially using an
-	//       unsupported/tested configuration
+func (iostat *Iostat) run(mts []plugin.Metric) ([]string, map[string]float64, error) {
 	// TODO: allow the path and/or name of the command to be overriden through the pluginConfigType
 
+	versionString := iostat.cmd.Exec("iostat", []string{"-V"})
+	version, err := iostat.parser.ParseVersion(versionString)
+	if err != nil {
+		return nil, nil, err
+	}
+	if version[0] < 10 || (version[0] == 10 && version[1] < 2) {
+		return nil, nil, fmt.Errorf("Iostat %d.%d.%d version (required >=10.2.0)", version[0], version[1], version[2])
```

Comment:
This error message could be a little bit more intuitive (e.g. `This plugin requires iostat in version 10.2.0 or newer (version present={version})`)